### ISSUE TITLE
4.0.2: AttributeError: module 'astral' has no attribute 'Location'

### ISSUE THREAD

**Issue by LordMike** [home-assistant/appdaemon#902]
Updated to resolve the leapday issue, and was met with this. This was also mentioned in #897, with a possible workaround that is fixing the Astral dependency to `1.10.1`.

```
Requirement already up-to-date: adutils>=0.4.9 in /usr/local/lib/python3.8/site-packages (from -r /conf/apps/ench/requirements.txt (line 1)) (0.4.10)
Requirement already up-to-date: adutils~=0.4.1 in /usr/local/lib/python3.8/site-packages (from -r /conf/apps/healthcheck/requirements.txt (line 1)) (0.4.10)
2020-02-29 15:11:46.928805 INFO AppDaemon: AppDaemon Version 4.0.2 starting
2020-02-29 15:11:46.928955 INFO AppDaemon: Python version is 3.8.2
2020-02-29 15:11:46.929058 INFO AppDaemon: Configuration read from: /conf/appdaemon.yaml
2020-02-29 15:11:46.929158 INFO AppDaemon: Added log: AppDaemon
2020-02-29 15:11:46.929262 INFO AppDaemon: Added log: Error
2020-02-29 15:11:46.929356 INFO AppDaemon: Added log: Access
2020-02-29 15:11:46.929460 INFO AppDaemon: Added log: Diag
2020-02-29 15:11:46.937156 WARNING AppDaemon: ------------------------------------------------------------
2020-02-29 15:11:46.937270 WARNING AppDaemon: Unexpected error during run()
2020-02-29 15:11:46.937357 WARNING AppDaemon: ------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/appdaemon/__main__.py", line 117, in run
    self.AD = ad.AppDaemon(self.logging, loop, **appdaemon)
  File "/usr/local/lib/python3.8/site-packages/appdaemon/appdaemon.py", line 188, in __init__
    self.sched = scheduler.Scheduler(self)
  File "/usr/local/lib/python3.8/site-packages/appdaemon/scheduler.py", line 54, in __init__
    self.init_sun()
  File "/usr/local/lib/python3.8/site-packages/appdaemon/scheduler.py", line 187, in init_sun
    self.location = astral.Location((
AttributeError: module 'astral' has no attribute 'Location'
```

**Comment by LordMike** [comment]
Sidenote: The 4.0.2 tag is also missing from dockerhub :)

**Comment by juan11perez** [comment]
facing the same error. 
reverting to 4.0.1 normalises operation

**Comment by valentingc** [comment]
Got the same issue here.

**Comment by acockburn** [comment]
I’ll be issuing a fix for this today or tomorrow

**Comment by acockburn** [comment]
Fixed on PyPi and on docker hub in v 4.0.3
